yaptide/static/openapi.yaml

```diff
@@ -877,10 +877,18 @@ paths:
   /results:
     description: Now it works properly only for simulations run on direct. Fetching results from batch will always end up with getting response with 400 status code.
     get:
```

Comment:
This description seems a bit nonsense. Moreover when clicking on "/results" in swagger I get:

![image](https://github.com/user-attachments/assets/4ba50bbc-0b55-4219-8041-e05c26803012)

[Screencast from 2024-10-13 21-04-08.webm](https://github.com/user-attachments/assets/dbba3c0a-7784-4ac3-9bc4-58a0affcfb23)

Reply:
Does this description appears somewhere on swagger UI ?

Reply:
I can't see the description on the ui, only the summary is visible. I have fixed this error by renaming "Estimators" object to "Estimator" in openapi.yaml.